docs/conversation.md

```diff
@@ -81,6 +81,7 @@
         `statusMessage` | string | v4 | | Optional: Only available for one-to-one conversations and when  `includeStatus=true` is set
         `participants` | array | v1 | v2 | **Removed**
         `guestList` | string | v1 | v2 | **Removed**
+        `timeToLive` | int | v4 | The time to live of messages in this chat. Zero if disabled. (only available with `time-to-live` capability)
```

Comment:
```suggestion
        `timeToLive` | int | v4 | | The time to live of messages in this chat. Zero if disabled. (only available with `time-to-live` capability)
```
Empty column is needed, so the Description is in the right column:

<img width="975" alt="2022-05-09 19_50_44-Implement time to live by vitormattos · Pull Request #7327 · nextcloud_spreed" src="https://user-images.githubusercontent.com/213943/167468253-9aa64fe1-e1f5-4da9-84e9-2de42f4fbc16.png">

Also it should be above `participants` so removed items are at the end.